controllers/numaresourcesoperator_controller.go

```diff
@@ -63,7 +63,7 @@ import (
 )
 
 const (
-	defaultNUMAResourcesOperatorCrName = "numaresourcesoperator"
+	DefaultNUMAResourcesOperatorCrName = "numaresourcesoperator"
```

Comment:
I'd move this const into `pkg/objectnames`. Let's try to avoid dependencies on the `controllers` package (which is another constraint `operator-sdk` is enforcing, inherited by `kubebuilder`)